src/workerd/api/sockets.c++

```diff
@@ -248,7 +250,7 @@ jsg::Ref<Socket> Socket::startTls(jsg::Lock& js, jsg::Optional<TlsOptions> tlsOp
   // to `setupSocket`.
   auto newTlsStarter = kj::heap<kj::TlsStarterCallback>();
   return setupSocket(js, kj::newPromisedStream(kj::mv(secureStreamPromise)), kj::mv(options),
-      kj::mv(newTlsStarter), true, kj::mv(domain), kj::mv(port));
+      kj::mv(newTlsStarter), true, kj::mv(domain), kj::mv(isFetchablePort));
```

Comment:
nit: No need to `kj::mv` a boolean :-) ... 

also boolean args are yucky ;-) (that's non-blocking tho)